.devcontainer/devcontainer.json

```diff
@@ -23,31 +25,35 @@
 		"python.linting.lintOnSave": true,
 		"python.linting.flake8Enabled": true,
 		"editor.formatOnSave": true,
-		"python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
-		"python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
-		"python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
+		"python.linting.banditPath": "/usr/local/bin/bandit",
+		"python.linting.flake8Path": "/usr/local/bin/flake8",
+		"python.linting.mypyPath": "/usr/local/bin/mypy",
 		"python.linting.pycodestylePath": "/usr/local/bin/pycodestyle",
 		"python.linting.pydocstylePath": "/usr/local/bin/pydocstyle",
 		"python.linting.pylintPath": "/usr/bin/pylint",
+		"terminal.integrated.defaultProfile.linux": "bash (login)",
 		"terminal.integrated.profiles.linux": {
 			"bash (login)": {
-				"path": "/bin/bash",
-				"args": [
-					""
-				]
+				"path": "bash"
 			}
 		}
 	},
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
 		"ms-python.python",
 		"ms-python.vscode-pylance",
-		"marklarah.pre-commit-vscode"
+		"marklarah.pre-commit-vscode",
+		"ms-toolsai.jupyter",
+		"ms-toolsai.jupyter-renderers",
+		"ms-toolsai.jupyter-keymap",
+		"jithurjacob.nbpreviewer"
 	],
+	// Mount to a volume if you'd like to persist data to your disk
+	// "mounts": ["source=<add your /path/on/host here>, target=/workspaces/hermes_core, type=bind"],
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
-	// Use 'postCreateCommand' to run commands after the container is created.
-	//"postCreateCommand": "bash -i -c 'pip3 install --user .[all]'",
+	// Hermes_core package is uninstalled from the container image after it is created to avoid circular import errors during development 
```

Comment:
Why was this an issue? Is hermes_core currently being uninstalled or are you changing it here so that it is not installed to begin with?

Reply:
It wasn't an issue; I just copied over the `.devcontainer.json` to keep them consistent with the hermes_core dev container config and accidentally replace this, I'll change this back. Good catch